intf_.mod

```diff
@@ -1129,8 +1131,8 @@ VERBATIM
   }
   if (!ifarg(iarg)) prfl=0; else { prfl=1;
     voi=vector_arg(iarg); 
-    if (flag==2.) { x=vector_newsize(voi,CTYPi); for (i=0;i<CTYPi;i++) x[i]=0;
-    } else x=vector_newsize(voi,sz); 
+    if (flag==2.) { x=vector_newsize((IvocVect*)voi,CTYPi); for (i=0;i<CTYPi;i++) x[i]=0;
```

Comment:
Shouldn't `voi` be declared as `IvocVect*` in such cases?

Reply:
you are right. I just fixed it.